package.json

```diff
@@ -13,8 +13,11 @@
   "scripts": {
     "start": "node index",
     "build": "grunt && handlebars client/views/ -e tpl -f client/js/lounge.templates.js",
-    "test": "HOME=test/fixtures mocha test/**/*.js && npm run lint",
-    "lint": "eslint . && stylelint \"**/*.css\"",
+    "test": "HOME=test/fixtures mocha test/**/*.js",
+    "lint": "npm run lint:js; npm run lint:css",
+    "lint:js": "eslint .",
+    "lint:css": "stylelint \"**/*.css\"",
+    "test-all": "npm run lint; npm run test",
```

Comment:
Yeah, I agree with this. Once this is done, I'll give my ok to this. Looks good.